Bio/PDB/PDBList.py

```diff
@@ -412,7 +415,13 @@ def update_pdb(self, file_format=None, with_assemblies=False):
                     print(f"Obsolete file {old_file} is missing")
 
     def download_pdb_files(
-        self, pdb_codes, obsolete=False, pdir=None, file_format=None, overwrite=False
+        self,
+        pdb_codes,
+        obsolete=False,
+        pdir=None,
+        file_format=None,
+        overwrite=False,
+        max_num_threads: Union[int, None] = None,
```

Comment:
I believe the convention is to use ``Optional[int]`` instead.

Or more simply, default to zero and use that as no limit?

Reply:
Might also not bother with adding type annotation, or do the whole function if you fancy?

Reply:
I decided to add type annotations to all the parameters, and I use Optional[int] now as you suggested.